region/brinstar/green/Green Brinstar Main Shaft.json

```
@@ -1604,6 +1605,39 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [13, 14],
+      "name": "Green Brinstar Main Shaft Bottom Ice Clip",
+      "requires": [
+        "Morph",
+        {"enemyDamage": {
+          "enemy": "Zeela",
+          "type": "contact",
+          "hits": 1
+        }},
+        "canTrickyUseFrozenEnemies",
+        "h_canNonTrivialCeilingClip",
+        "SpringBall",
+        {"or": [
+          "Bombs",
+          {"ammo": {"type": "PowerBomb", "count": 2}}
+        ]}
```

I'm thinking this is a good indicator of if you need to bring down the higher Zeela. It's looking like Supers dont help much to speed it up, since you can't use them for the bottom long descent.
```suggestion
        ]},
        {"or": [
          "canBePatient",
          {"obstaclesNotCleared": ["A"]}
        ]}
```